torch/_torch_docs.py

```diff
@@ -9214,8 +9214,22 @@ def merge_dicts(*dicts):
 .. math::
     \text{{out}}_{{i}} \sim \mathcal{{N}}(0, 1)
 
+For complex dtypes, the tensor is i.i.d. sampled from a `complex normal distribution`_ with zero mean and
+unit variance as
+
+.. math::
+    \text{{out}}_{{i}} \sim \mathcal{{CN}}(0, 1)
+
+This is equivalent to separately sampling the real :math:`(\operatorname{{Re}})` and imaginary
+:math:`(\operatorname{{Im}})` part of :math:`\text{{out}}_i` as
+
+.. math::
+    \operatorname{{Re}}(\text{{out}}_{{i}}) \sim \mathcal{{N}}(0, \frac{{1}}{{2}}),\quad
```

Comment:
```suggestion
    \operatorname{{Re}}(\text{{out}}_{{i}}) \sim \mathcal{{N}}\Big(0, \frac{{1}}{{\sqrt{{2}}}}\Big),\quad
```
Same below

Reply:
I can fix the parens, but the formula should be correct according to [Wikipedia](https://en.wikipedia.org/wiki/Complex_normal_distribution). We also list the other equations as `N(mean, variance)`, so `N(mean, std)` would probably be confusing here.

Reply:
Ah, right, we are showing the variance. Yep, then it's right.